src/app/core/cookie-agreement.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CookieAgreementService {
+  cookiesAgreement = false;
+  showCookieBanner = true;
+
+  constructor() {}
+
+  getCookieAgreementVal(): boolean {
+    this.checkCookieAgreement();
+    return this.cookiesAgreement;
+  }
+
+  getShowCookieBannerVal(): boolean {
+    this.checkShowCookieBanner();
+    return this.showCookieBanner;
+  }
+
+  /**
+   * @Check_Functions
```

Comment:
What does this do?

Reply:
I just changed it to display a comment that explains the checking functions purpose
"The checking functions are helper functions to check the values stored in users localStorage."